### PR TITLE
docs: Http

### PR DESCRIPTION
**Pull request by Ombuweb**

### CODE REVIEW

File: content/guide/core/http.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+---
+
+#### toJSON()
+
+```ts
+response.content?.toJSON(encoding)
+```
+
+Gets the response body as JSON object. The `encoding` is an optional of type `'UTF8' | 'GBK'`
```
Comment (ashutosh887) [inline content/guide/core/http.md]: We can keep `UTF8` & `GBK` as external links and on clicking, the CTA could be more detail/some docs page for encoding types

**Review by ashutosh887 — APPROVED** [review]
Rest of the things LGTM

**Comment by ashutosh887** [comment]
@NathanWalker Added some comments!

File: content/guide/core/http.md (hunk 1 above)
Comment (NathanWalker) [inline content/guide/core/http.md]:
```suggestion
Gets the response body as JSON object. The `encoding` is optional of type `'UTF8' | 'GBK'`.
```

File: content/guide/core/http.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
+---
+title: Http
+---
+<!-- TODO: Add links -->
+This class allows you to make http requests.
```
Comment (NathanWalker) [inline content/guide/core/http.md]: `Http` provides utility methods to make all sorts of http networking requests.

Comment (NathanWalker) [inline content/guide/core/http.md]:
```suggestion
Http provides utility methods to make all sorts of http networking requests.
```

File: content/guide/core/http.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+)
+```
+
+### GET request with a response body as a JSON
+
+Use the [getJSON()](#getjson) method for a GET request with a response as a JSON:
+
+```ts
+Http.getJSON('https://httpbin.org/get').then(
+  (result: any) => {
```
Comment (NathanWalker) [inline content/guide/core/http.md]:
```suggestion
  (result) => {
```

File: content/guide/core/http.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+```ts
+Http.getString('https://httpbin.org/get').then(
+  (result: string) => {
+    // do something with the string response
+  },
+  e => {}
+)
+```
+
+### GET request with a response body as a JSON
```
Comment (NathanWalker) [inline content/guide/core/http.md]:
```suggestion
### GET request with a response body as JSON
```

**Review by NathanWalker — APPROVED** [review]

**Pull request merged**